utils/python/CIME/SystemTests/system_tests_common.py

```diff
@@ -158,7 +158,7 @@ def run(self):
             self._test_status.set_status(RUN_PHASE, status, comments=("time=%d" % int(time_taken)))
 
         # We only return success if every phase, build and later, passed
```

Comment:
Thanks a lot, @jgfouca - I really appreciate it.

However, you should now remove this comment on line 160.